fix shebang in map-variants.py to use 'python3' command available via $PATH

author: Kenneth Hoste (HPC-UGent)
diff -ru xkeyboard-config-2.29.orig/rules/compat/map-variants.py xkeyboard-config-2.29/rules/compat/map-variants.py
--- xkeyboard-config-2.29.orig/rules/compat/map-variants.py	2020-01-31 01:29:55.000000000 +0100
+++ xkeyboard-config-2.29/rules/compat/map-variants.py	2020-10-13 18:09:47.347929930 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
 import argparse
 import re
